examples/forum/forum_util.py

```diff
@@ -0,0 +1,88 @@
+import json
+
+# each value in input is an array even if there is only one element
+def jsonify_urllib_params(input_dict):
+	new_dict = {}
+	for key in input_dict:
+		# filter quote from key and ignore metadeta (key starting with _)
+		nk = filter_out_char(key, "\"")
+		if key[0] != "_":
+			val = input_dict[key]
+			if len(val) == 1:
+				val = val[0]
+			new_dict[nk] = val
+	return json.dumps(new_dict)
+
+# I could use the built-in filter function, but this is a little more clear
+def filter_out_char(in_str, rem_ch):
```

Comment:
I think you can actually use the replace() function here: `'Test string "with" quotes'.replace('"', '')`

Reply:
I decided I don't actually need this function for anything.